Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/MainActivity.java

```diff
@@ -205,25 +235,72 @@ public void onLoadFinished(Loader<Cursor> loader, Cursor data)
 		int id = loader.getId();
 		Log.d("DolphinEmu", "Loader finished with id: " + id);
 
-		// TODO When each platform has its own tab, this should just call into those tabs instead.
+		PlatformGamesFragment fragment = null;
 		switch (id)
 		{
-			case LOADER_ID_GAMES:
-				mAdapter.swapCursor(data);
-				// TODO Play some sort of load-finished animation; maybe fade the list in.
+			case LOADER_ID_GAMECUBE:
+				fragment = getPlatformFragment(Game.PLATFORM_GC);
+				break;
+
+			case LOADER_ID_WII:
+				fragment = getPlatformFragment(Game.PLATFORM_WII);
+				break;
+
+			case LOADER_ID_WIIWARE:
+				fragment = getPlatformFragment(Game.PLATFORM_WII_WARE);
 				break;
 
+			// TODO case LOADER_ID_ALL:
+
 			default:
 				Log.e("DolphinEmu", "Bad ID passed in.");
```

Comment:
A break in the default case? Wouldn't that be unnecessarily verbose? It wouldn't any any functional effect.